settings.js

```diff
@@ -184,6 +184,7 @@ function envValueBoolean(varName, defaultValue) {
 
 settings.findHomePath = () => {
 	let envVars = [
+		'XDG_DATA_HOME',
```

Comment:
`$HOME` and `$XDG_DATA_HOME` are not equivalent.

from the [spec](https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html):

> $XDG_DATA_HOME defines the base directory relative to which user-specific data files should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used.